src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/SubmitToSaboModal.tsx

```diff
@@ -114,6 +114,32 @@ const SubmitToSaboModal = ({ open, setOpen, reimbursementRequest }: SubmitToSabo
           <DetailDisplay label="Expense Type" content={`${expenseType.code} - ${expenseType.name}`} />
         </Grid>
       </Grid>
+      <Grid container spacing={1} sx={{ marginTop: 2 }}>
+        <Grid item xs={3}>
+          <DetailDisplay label="Treasurer" content="" />
+        </Grid>
+        <Grid item xs={6}>
+          <div>
+            <Typography sx={{ fontWeight: 'normal', display: 'inline' }}>Brody Pearlman</Typography>
+          </div>
+          <div>
+            <Typography sx={{ fontWeight: 'normal', display: 'inline' }}>pearlman.edu@northeastern.edu</Typography>
+          </div>
+        </Grid>
+      </Grid>
+      <Grid container spacing={1} sx={{ marginTop: 2 }}>
+        <Grid item xs={3}>
+          <DetailDisplay label="Club Adivsor" content="" />
+        </Grid>
+        <Grid item xs={6}>
+          <div>
```

Comment:
Generally, avoiding using \<div>, just because its a native HTML element and we prefer to stay consistent with fancy React elements. If you want to enforce these being on top of each other you could use a \<Box> (basically the same as a div), a \<Stack> (stacks things vertically), or you could create another \<Grid> for the text. Whatever you prefer really.